dataactcore/utils/report.py

```diff
@@ -1,18 +1,26 @@
 import itertools
 from operator import attrgetter
 
-from dataactcore.models.lookups import FILE_TYPE
+from dataactcore.models.lookups import FILE_TYPE, FILE_TYPE_DICT_NAME_LETTER
 
 
 def report_file_name(submission_id, warning, file_type, cross_type=None):
     """Format the csv file name for the requested file.
     @todo: unify these file names"""
     if cross_type:
         report_type_str = 'warning_' if warning else ''
-        return "submission_{}_cross_{}{}_{}.csv".format(submission_id, report_type_str, file_type, cross_type)
+        file_letters = ''
+        if file_type in FILE_TYPE_DICT_NAME_LETTER and cross_type in FILE_TYPE_DICT_NAME_LETTER:
```

Comment:
Why are these `if` statements? In what situation would we be passing a file type that's not in our list?

Reply:
There's a test that does this. We can update the test? Or handle cases like the test.
https://github.com/fedspendingtransparency/data-act-broker-backend/blob/8302d223d1da63bd7d8a34ab7958edce90388f3a/tests/unit/dataactbroker/test_file_handler.py#L439-L450

Reply:
I don't see where we could possibly have an invalid file type actually within the code. This seems like an old test if anything.